src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1318,6 +1323,18 @@ private function exportParameters(array $parameters, string $path = '', int $ind
 
     private function endClass(): string
     {
+        if ($this->addThrow) {
+            return <<<'EOF'
+
+    protected function throw($message)
```

Comment:
I don't see the advantage of creating a method for this.

Reply:
We need to throw inline, ie in expressions, but the `throw` keyword is not allowed there.